tests/consensus_tests/test_partial_snapshot.py

```
@@ -0,0 +1,173 @@
+    if bootstrap_points > 0:
+        upsert(peer_url, bootstrap_points)
+
+def recover_collection(peer_url: str, recover_from_url: str):
+    snapshot_url = create_collection_snapshot(recover_from_url)
+    recover_collection_snapshot(peer_url, snapshot_url)
+
+    assert_point_consistency(peer_url, recover_from_url)
+
+def create_collection_snapshot(peer_url: str):
+    resp = requests.post(f"{peer_url}/collections/{COLLECTION}/snapshots")
+    assert_http_ok(resp)
```

Ruff (0.8.2)

`assert_http_ok` may be undefined, or defined from star imports

(F405)